### PR TITLE
chore(docs): Remove broken docker deploy button

### LINKED ISSUE #6556: The image in readme broken
**Describe the bug**
The image in readme is broken

**To Reproduce**
1. Go to Readme.md
3. Scroll down to Installation 
4. See the broken image

**Additional context**
Screenshot 2019-10-28 at 8 07 39 AM

### PR DESCRIPTION
**Pull request by Swap76**
Fixes #6556 

#### Short description of what this resolves:

The image of docker in the Readme.md file is broken 

#### Changes proposed in this pull request:

- Added new SVG image for Docker

Screenshot 2019-10-30 at 3 45 57 PM

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ The Open Event Server can be easily deployed on a variety of platforms. Detailed
 
 One-click Heroku and Docker deployments are also available:
 
-[![Deploy to Docker Cloud](https://files.cloud.docker.com/images/deploy-to-dockercloud.svg)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy to Docker Cloud](https://upload.wikimedia.org/wikipedia/commons/thumb/4/4e/Docker_%28container_engine%29_logo.svg/150px-Docker_%28container_engine%29_logo.svg.png)](https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server)  [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
```
Comment (iamareebjamal) [inline README.md]: Gives 404, so either this should be removed or fixed

Comment (Swap76) [reply README.md]: @iamareebjamal I am seeing the image on GitHub also in PR 
Screenshot 2019-10-30 at 7 36 51 PM

Comment (Swap76) [reply README.md]: > https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server

@iamareebjamal The link is the same as you have sent above for after the click

Comment (Swap76) [reply README.md]: @iamareebjamal I am not getting what you are telling sorry for that as the link "https://cloud.docker.com/stack/deploy/?repo=https://github.com/fossasia/open-event-server" opens **hub.docker.com** for me

Comment (iamareebjamal) [reply README.md]: "And it gives 404, so it doesn't work."

Can you deploy to docker cloud through the link? 

Does it show anything else than `404 Page not found`?

If not, then the link doesn't work. There is no use of a button which doesn't work. Either it should be fixed so clicking on it deploys to docker cloud or removed

Comment (Swap76) [reply README.md]: @iamareebjamal I got it now what you are trying to say will remove it ASAP

**Comment by Swap76** [comment]
@iamareebjamal I have removed the link and also changed the line of the explanation above it accordingly

**Pull request merged**